.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
- [ **OPTIONAL** ] I suggested you also add this `.vscode/` directory in your `.gitignore` file as it contains nonrelated files of the project so it can be ignored by Git and not appear on GitHub.

Reply:
@tresorsawasawa I have added .vscode/ directory in my .gitignore file